apps/task/models.py

```diff
@@ -62,6 +75,26 @@ class Task(BaseModel):
     def __str__(self):
         return self.title
 
+    @staticmethod
+    def get_for(user):
+        task_group = TaskGroup.get_for(user)
+        return Task.objects.filter(
+            user=user,
+            task_group__in=task_group
+        ).distinct()
+
+    def duration_task(self, user):
+        task = Task.objects.get(id=self.id)
+        duration = TimeEntry.objects.filter(
+            user=user,
+            task=task
```

Comment:
So you can do something like, `self.time_entries.filter( ...`. Something like this...